src/common.h

```diff
@@ -354,6 +354,12 @@ enum class SymbolBinding {
   Local = 2,
 };
 
+/* matches binary format, do not change */
+enum class BranchHintKind : uint32_t {
```

Comment:
Maybe elaborate on what this class represents?   Should we maybe create a new header called `code_annotations.h`?   We don't want folks reading this to think it relates to branches in the wabt codebase itself.

Reply:
good catch. It is a remnant of a previous implementation, but it is not used anymore. I will remove it.